pkg/utils/common.go

```diff
@@ -121,6 +121,20 @@ func FetchAllDoTables(db *sql.DB, bw *filter.Filter) (map[string][]string, error
 
 // FetchTargetDoTables returns all need to do tables after filtered and routed (fetches from upstream MySQL)
 func FetchTargetDoTables(db *sql.DB, bw *filter.Filter, router *router.Table) (map[string][]*filter.Table, error) {
+	failpoint.Inject("mockGenerateExtraArgs", func(_ failpoint.Value) {
```

Comment:
can we move this `mockGenerateExtraArgs ` to the caller (`fetchMyDumperDoTables`) in _mydumper/util.go_?